### DISCUSSION TITLE
LOCAL BRANCH OUTLINE VIEW

### DISCUSSION THREAD

**Discussion by Illegit** [freeplane/freeplane#302] [Help]
![image](https://user-images.githubusercontent.com/62674157/162582765-acf63f64-6fea-45ee-a2dd-a8e12ae118ed.png)

Hi, currently I saw only two global ways to view branches: normal view map or outline view map

I remember some time ago there was in a forum a discussion about having the possibility to activate different shapes in local branches.
Here is a simple scheme, my question: is there a possibility, even with a script (if not in normal FP functions), to activate a local outline view of a branch?
Thank you
illegit

**Comment by macmarrum** [comment]
@Illegit indeed, the topic was discussed and the concept was evaluated: https://sourceforge.net/p/freeplane/discussion/758437/thread/2b55f79c81/#1845

It is possible to manually place nodes one under another. I am guessing that a script could be created to automate it.

**Reply by Illegit** [reply]
now  I saw it. Probably it is necessary to create a script to be loaded in a node of a specific style...
illegit
